### ISSUE TITLE
Windows version information improvements

### ISSUE THREAD

**Issue by tresf** [qzind/tray#1028]
Internally, we get the Windows version using a registry key:

```reg
HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion (ReleaseId)
```

... however this version information is no longer valid.  See also https://ss64.com/nt/ver.html.

Objectives:

1. Make the Windows version on the version information page show build information.
1. Make the Windows version on the version information page display the friendly build number (e.g. 21H2).

Product name | Major version.build | Release ID | Version
-- | -- | -- | --
Windows 10 version 21H2 (October 2021 Update) | 10.0.19044 | 2009 | 21H2
Windows 11 version 21H2 (October 2021 Update) | 11.22000.19411.22000.1042 | 2009 | 21H2

**Comment by Vzor-** [comment]
pr #1031 

for my own reference: https://www.technlg.net/windows/windows-version-in-registry-key/ https://www.thecliguy.co.uk/2018/10/07/windows-version-and-build-number/

**Comment by thecliguy** [comment]
@Vzor- Hello, I noticed that you linked to my blog post on [Windows Version and Build Number](https://www.thecliguy.co.uk/2018/10/07/windows-version-and-build-number).

I originally wrote that post in 2018 and therefore the PowerShell code under `Windows 10 / Windows Server 2016 (and beyond?)` suffered from the problem of returning `2009` for `20H2` and all subsequent versions of Windows. 

I've now updated the code so it uses `DisplayVersion` if present in the registry, else it uses `ReleaseId`. This approach should mean that the code works with versions released before and after `20H2`. Just to make sure, I dug out a very old laptop with `1809` on it and verified that the code worked by using `ReleaseId` in the absence of `DisplayVersion`.

**Comment by Vzor-** [comment]
@thecliguy Thanks for reaching out! Your website has always been a great resource. I wish Microsoft would 
release some official documentation on how to interpret and retrieve version numbers. It seems like the version semantics have changed more often than the version itself.

I was checking to see if `ReleaseId == '2009'` then falling back to `DisplayVersion`, but I think I like your solution more of just checking to see if DisplayVersion exists.

**Comment by tresf** [comment]
Closed via #1031
